submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,146 @@
+
+* **Nuclear plants are bad for the game performance (UPS)** – While it is true that nuclear power performs unarguably worse than solar power at the massive megabase scale, there is no noticeable UPS impact from small and medium sized nuclear plants. We explore the details of this in the section "Nuclear and UPS".
+
+With these concerns addressed, let us look into the DIY of nuclear power.
+
+### Designing your own Nuclear Plants
+
+#### How do I know how many components I need?
+
+When starting out, it makes sense to start with single reactor, or perhaps a 1x2 (1 row of 2 reactors) layout. The [wiki tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) has you covered for the basics. After that, the size of your design is based on your power needs. You can arrange your reactors however you want, but to get the neighbor bonus between any two reactors, they must be sharing a full edge. After designing your setup, the easy way to calculate the total power output for it is by making a single reactor equivalent (SRE) diagram. This process works in three steps. We can go over the steps with an example, using a setup that is among my favorites:
+
+<!--VISUAL 3: A design I made with 8 reactors in a 3x3 grid. It is the most powerful 3x3 setup (1040MW output) that has at least 1 face of each reactor exposed for automated refueling.-->
+
+1) Draw a copy of your reactor arrangement and to each reactor assign a value of 1. Any empty spaces receive 0:
+
+```
+[1][1][1]
+[1][1][1]
+[1][0][1]
+```
+
+2) For each reactor, count the number of neighbors it has (they must be aligned exactly side by side) and add it to the 1:
+
+```
+[1+2][1+3][1+2] --> [3][4][3]
+[1+3][1+3][1+3] --> [4][4][4]
+[1+1][0+0][1+1] --> [2][0][2]
+```
+
+3) Sum all the numbers to get the total number of single reactor equivalents. Multiply this total by 40 to get your heat output in MW:
+
+```
+(3+4+3) = 10
+(4+4+4) = 12
+(2+0+2) = 4
+
+10 + 12 + 4 = 26 single reactor equivalents for this setup.
+26 * 40MW = 1040MW power output
+```
+
+After determining your total power output, you can decide whether you have enough reactors or a sufficiently optimal arrangement of them. However, this is only the beginning of the design process. Next, we have to turn reactor heat into steam with heat exchangers, which we will need to supply with water from offshore pumps. Then, the steam needs to be consumed by steam turbines.
+
+Since the reactors produce their power indirectly, in the form of heat, we need to harness this power using heat exchangers. Each heat exchanger consumes up to 10MW in terms of heat, and it is best to have enough of them to harness the reactors' maximum output. Hence the number of heat exchangers needed is the total power output in MW divided by 10. So, in our example we have 1040MW/10MW = 104 heat exchangers.
```

Comment:
```suggestion
Since the reactors produce their power indirectly, in the form of heat, we need to harness this power using heat exchangers. Each heat exchanger consumes up to 10 MW in terms of heat, and it is best to have enough of them to harness the reactors' maximum output. Hence the number of heat exchangers needed is the total power output in MW divided by 10. So, in our example we have 1040 MW/10 MW = 104 heat exchangers.
```